libs/design/TableSkeleton/TableSkeleton.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+
+import { TableCell } from '../Table/TableCell';
+import { TableRow } from '../Table/TableRow';
+
+export const TableSkeleton = () => {
+  const numberOfSkeletonRows = 10;
+  const numberOfSkeletonCells = 4;
+
+  const skeletonRows = Array.from(
```

Comment:
If we memoize this one react can skip re-creating all of these components on each render